app/templates/course/stage/instructions.hbs

```diff
@@ -41,6 +41,7 @@
         @repository={{@model.activeRepository}}
         @courseStage={{@model.courseStage}}
         @shouldRecommendLanguageGuide={{this.shouldShowLanguageGuide}}
+        @shouldShowSolution={{this.shouldShowSolution}}
```

Comment:
```suggestion
        @shouldShowSolution={{this.shouldShowStage2Solution}}
```

(related to above comment)

The arg name is fine, since that's on `SecondStageInstructionsCard`.